packages/EasyCore/src/Interfaces/DatabaseEntityInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Interfaces;
+
+interface DatabaseEntityInterface
+{
+    public function getId(): string;
```

Comment:
Let's left this interface empty. In some projects (like Points) we have a different signature of this method.

Reply:
Done